templates/website/confirm-accept.html

```diff
@@ -18,19 +18,25 @@
             </nav>
         </div>
         <a href="/" class="wtt-logo"><img src="/static/img/logo.png" width="250" height="70" alt="WriteToThem"></a>
-        <div style="text-align:center;">
-            <h2>All done! Your message is on its way.</h2>
-            <p class="subtitle">If, for some reason, we couldn&rsquo;t send your message
-                to your representative, we will email you to tell you. Otherwise, you can sit
+        <div style="background-color: rgba(0, 0, 0, 0.5);padding-top:10px;padding-bottom:10px">
+        <div style="text-align:center;" >
+            <h2 style="font-size: 2em;">All done! Your message is on its way.</h2>
```

Comment:
If we get rid of the "all done", it'll fit on one line?

Reply:
Yep!